Src/WitsmlExplorer.Api/Jobs/JobInfo.cs

```diff
@@ -73,6 +74,17 @@ public JobStatus Status
             }
         }
 
+        public string LinkType
```

Comment:
Rename to ReportType. Also, it makes more sense to add an enum with the options "File" and "Report".

Reply:
done